infra/blueprint-test/pkg/tft/terraform.go

```diff
@@ -40,6 +40,7 @@ const setupKeyOutputName = "sa_key"
 type TFBlueprintTest struct {
 	discovery.BlueprintTestConfig                          // additional blueprint test configs
 	name                          string                   // descriptive name for the test
+	saKey                         string                   // optional setup encoded sa key
```

Comment:
If we are specifying an explicit SA key, I would expect it to be already decoded (i.e not b64 encoded) since thats how you would download/generate it via gcloud. WDYT?

Reply:
I agree.

My script was already generating base64 so I ended up skewing it.